src/Optimization/hiopKKTLinSysSparse.cpp

```diff
@@ -612,13 +618,23 @@ namespace hiop
         }
 
         if( (nullptr == linSys_ && linear_solver == "auto") || linear_solver == "strumpack") {
-          //ma57 is not available or user requested strumpack
+          //ma57 and pardiso are not available or user requested strumpack
 #ifdef HIOP_USE_STRUMPACK              
           linSys_ = new hiopLinSolverIndefSparseSTRUMPACK(n, nnz, nlp_);
           actual_lin_solver = "STRUMPACK";
 #endif  // HIOP_USE_STRUMPACK        
         }
 
+        if( (nullptr == linSys_ && linear_solver == "auto") || linear_solver == "ginkgo") {
+          //ma57, pardiso and strumpack are not available or user requested ginkgo
+#ifdef HIOP_USE_GINKGO
+          hiopLinSolverSymSparseGinkgo *p = new hiopLinSolverSymSparseGinkgo(n, nnz, nlp_);
+          p->setFakeInertia(neq + nineq);
+          linSys_ = p;        
```

Comment:
"Fake" inertia should not be used anymore:
```suggestion
          linSys_ = new hiopLinSolverSymSparseGinkgo(n, nnz, nlp_);
```